README.md

```diff
@@ -1 +1,116 @@
 # python-esiclient
+
+`python-esiclient` extends the OpenStack CLI to provide simplified user workflows, encapsulating multiple OpenStack CLI commands into a single CLI command.
+
+## `openstack esi node network <command>`
+
+These commands manage network connections to nodes.
+
+### `openstack esi node network list`
+
+List node/network attachments.
+
+```
+   openstack esi node network list
```

Comment:
I would drop the initial indentation in the code samples in this document. It just leads to extra whitespace if someone tries to copy-and-paste it somewhere.

Reply:
Done!